mautic/mautic
# php-cs-fixer 2.2 changes

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | N
| Related user documentation PR URL | N/A
| Related developer documentation PR URL | N/A
| Issues addressed (#s or URLs) | #5010
| BC breaks? | Nil
| Deprecations? | Nil

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
1) use ``composer update`` to first update ``composer.lock`` to whatever it should have currently based on the existing ``composer.json``. This helps to make it clear what are just existing minor related package updates that are waiting to happen anyway.
2) bump ``php-cs-fixer`` to version 2.2.* (note this is the highest version that still melds with PHP5.6 and all the other interacting dependencies that ``composer`` calculates) and re-generate ``composer.lock``
3) update ``.php_cs`` to use the ``php-cs-fixer`` v2 syntax to set up the equivalent code standard checks that were being done with v1.
4) run ``php-cs-fixer`` to make the "fixes" it cares about.
5) review those "fixes" and make a few manual changes to line things up, or remove duplicated stuff...

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. run ``bin/php-cs-fixer fix -v --dry-run --diff`` in a development environment
2. see that it complains:
```
You are running PHP CS Fixer v1, which is not maintained anymore. Please update to v2.
```
The ``php-cs-fixer`` project would like users to get moving and move to v2.

#### Steps to test this PR:
1. run ``bin/php-cs-fixer fix -v --dry-run --diff`` in a development environment
2. see that the message no longer happens
3. let Travis-CI do its thing and see that it passes
